frontend/src/components/Table/HorizontalTable.tsx

```diff
@@ -62,51 +75,95 @@ export function HorizontalTable({ header, content }: HorizontalTableProps) {
     setRowsPerPage(parseInt(event.target.value, 10));
     setPage(0);
   };
+
+  const colSpan =
+    expandElement || optionElement ? header.length + 1 : header.length;
+
   return (
     <>
-      <thead className="border-b border-black/10 bg-second-accent font-bold text-text">
+      <thead className="border-b-2 border-accent font-bold text-text">
         <tr>
           {header.map((item) => (
-            <th
-              key={item}
-              onClick={() => handleSort(item)}
-              className="cursor-pointer p-3 text-left"
-            >
-              <div className="flex justify-between">
-                <p>
-                  {item}
+            <th key={item} className="cursor-pointer py-1 text-left">
+              <Button
+                fullWidth
+                className="text-left"
+                onClick={() => handleSort(item)}
+              >
+                <div className="flex justify-between">
+                  <p>{item}</p>
                   {sortBy === item ? (sortOrder === "asc" ? " ▲" : " ▼") : ""}
```

Comment:
Please move line 95 inside the `<p>` tag. This way we won't need to update some of the test cases.

Reply:
<img width="431" alt="Screenshot 2023-11-25 at 23 32 09" src="https://github.com/ducanhpham0312/zeevision/assets/113171462/9a4a0b26-8303-4e79-907d-5ec26b3f5618">

I was going for this design (where the sorting arrow indicator is placed at the opposite side as the header text), could it be styled somehow to still achive the design but have the {item} and the arrow icon in the same element?

Reply:
Hmm I guess you can keep this change then. I don't see any other way except that it should be in separate elements..